index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Vicente Flores Portafolio</title>
+    <meta charset="UTF-8" />
+    <meta name="description" content="This is just me and my experience" />
+    <meta name="author" content="Vicente Flores" />
+  </head>
+
+  <body>
+    <h1>Vicente Flores</h1>
+
+    <section id="about">
+      <h2>About</h2>
+      <p>This is a paragraph about me. Here’s more info about me.</p>
```

Comment:
The apostrophe is considered a special character - that is what AI reviewer is mentioning.   I found thi https://www.delftstack.com/howto/html/html-apostrophe/